### DISCUSSION TITLE
fluctuation in numbers

### DISCUSSION THREAD

**Discussion by AugustQu** [cyberman54/ESP32-Paxcounter#975] [Show and tell]
Hi,

last friday I used the PaxCounter for a real-life-test. On the location there were about 150 people (my guess). Here are some of the results:

18:40:00 | 75 |  
18:41:00 | 87 |  
18:42:00 | 113 |  
18:43:00 | 70 |  
18:44:00 | 79 |  
18:45:00 | 100 |  
18:46:00 | 99 |  
18:47:00 | 85 |  
18:48:00 | 88 |  
18:49:00 | 83 |  
18:50:00 | 124 |  

As you can see the numbers go from 70 to 124. On the place of the event the number of people was stable (maybe 3 to 5 persons come and go and these 10 minutes). As you can see on 18:42 the number fell from 113 to 70 and jumped up to 100 3 minutes later - but there was not such a fluctuation on the location.

What can we do to get more accurate numbers? Increase the interval for counting?

AugustQ

**Comment by cyberman54** [comment]
Don't expect absolute numbers, since we count random MACs, not static ones. But you can average the results, and see if and when and how much crowds increase and decrease.

You can count static MACs instead, if you adapt the code. But then, in many case, you will grab personal data, as well as unheaded devices like Wifi routers etc.

**Comment by cyberman54** [comment]
To get more stable (means more averaged) results, you may try to increase scan time (e.g. 2 - 5 mins) and reduce rssi sensitivity.

**Comment by AugustQu** [comment]
I had the same idea: increase the scan time.

I will try it and report my progress.

**Comment by AugustQu** [comment]
so far I had no success.

I tried to clear the NVRAM (as described in the docs). Here are the results:

pio run -t erase
******************************************************************************************************************************************************
Obsolete PIO Core v4.3.4 is used (previous was 6.1.7)
Please remove multiple PIO Cores from a system:
https://docs.platformio.org/page/faq.html#multiple-pio-cores-in-a-system
******************************************************************************************************************************************************
Processing usb (framework: arduino; board: esp32dev; platform: espressif32@6.3.0)
------------------------------------------------------------------------------------------------------------------------------------------------------
PlatformManager: Installing espressif32 @ 6.3.0
espressif32 @ 6.3.0 is already installed
Error: Unknown development platform 'espressif32'

sudo pio run -t erase
Processing usb (framework: arduino; board: esp32dev; platform: espressif32@6.3.0)
------------------------------------------------------------------------------------------------------------------------------------------------------
PlatformManager: Installing espressif32 @ 6.3.0
Error: Detected unknown package 'espressif32'
august@August-Ryzen:~/Arduino/Projects/ESP32-Paxcounter$ 

So what can I do?

**Reply by cyberman54** [reply]
Looks like you a have problem with your platformio toolchain, i am sorry, i can't provide support for this iusse.

Workaround: if you increase the paxcounter version number in platformio.ini, this will trigger a device NVRAM reset on first run of the updated device.

**Comment by AugustQu** [comment]
workaround: from 3.6.0 to 3.6.1 ?

OK, I will try this. Thanks for the tip.

**Reply by cyberman54** [reply]
3.6.0 -> 3.6.1 is okay.
use any decimal format like "a.b.c"

**Comment by AugustQu** [comment]
did it, worked. Thanks for the tip.

Coming friday I will make another real-life-test. Depending on the weather there will be >200 people on this place. I will report on the results.

**Reply by Neogene** [reply]
Hi how went the scanning? Which settings you used?
